manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -462,6 +462,13 @@ properties:
         secret: (( grab secrets.uaa_clients_firehose_password ))
         scope: openid,oauth.approvals,doppler.firehose
         authorities: oauth.login,doppler.firehose
+      datadog-nozzle:
```

Comment:
That has been added by mistake, it should not be here. 

I will fix it in a different PR.